package.bzl

```diff
@@ -47,9 +47,9 @@ def rules_typescript_dependencies():
     _maybe(
         http_archive,
         name = "io_bazel_rules_webtesting",
-        urls = ["https://github.com/bazelbuild/rules_webtesting/archive/89219129b13ec0f2cd6e65eff9113948e84e1856.zip"],
-        strip_prefix = "rules_webtesting-89219129b13ec0f2cd6e65eff9113948e84e1856",
-        sha256 = "71bcc2dc0d1d9c9e4f032dd2c0983fa73cb90adb48e86453f78071af8b2fc7ec",
+        urls = ["https://github.com/filipesilva/rules_webtesting/archive/d459ccfe31478bdddc71afe63dd9688399382ea0.zip"],
```

Comment:
This PR is using my forked version of rules_webtesting to test the chrome fix. Needs to be changed back to the canonical repo once https://github.com/bazelbuild/rules_webtesting/pull/303 is merged.